articles/web-application-firewall/waf-sentinel.md

```diff
@@ -79,6 +79,12 @@ To enable log analytics for each resource, go to your individual Azure Front Doo
 1. Once finished configuring individual WAF resources, select the **Next steps** tab. Select one of the recommended workbooks. This workbook will use all log analytic data that was enabled previously. A working WAF workbook should now exist for your WAF resources.
 
    :::image type="content" source="media//waf-sentinel/waf-workbooks.png" alt-text="WAF workbooks" lightbox="media//waf-sentinel/waf-workbooks.png":::
+   
+## Automated Detection and Response for Azure WAF with Sentinel
+
+There are built in detection query templates for SQLi and XSS attacks in Sentinel for Azure WAF. These templates can be found under the Analytics tab in the 'Rule Templates' section of Sentinel. By using these templates, we can create analytic rules that detect specific type of attack patterns in the WAF logs and further notify the security analyst by creating an incident. The automation section of these rules can help us respond to this incident by blocking the source IP of the attacker on the WAF Policy by utilizing a Playbook. This [Playbook](https://github.com/Azure/Azure-Network-Security/tree/master/Azure%20WAF/Playbook%20-%20WAF%20Sentinel%20Playbook%20Block%20IP%20-%20New) can be deployed from the Net Sec Github Repository. 
```

Comment:
Some suggesting changes

"Automatically detect and respond to threats"

You can use Sentinel analytics rules to automatically detect security attacks based on WAF logs, create security incident, and respond to security threats using playbooks. Learn more. [Use playbooks with automation rules in Microsoft Sentinel](https://learn.microsoft.com/en-us/azure/sentinel/tutorial-respond-threats-playbook?tabs=LAC) 
Azure WAF also comes in with built-in Sentinel detection rules templates for SQLi, XSS and Log4J attacks. These templates can be found under the Analytics tab in the 'Rule Templates' section of Sentinel. You can use these templates or define your own templates for attack detection based on WAF logs. 

<Insert screenshot of WAF rules templates>

The automation section of these rules can help you automatically respond to the incident by running a playbook  An example of such a playbook to respond to attack can be found in network security GitHub repository [here] https://github.com/Azure/Azure-Network-Security/tree/master/Azure%20WAF/Playbook%20-%20WAF%20Sentinel%20Playbook%20Block%20IP%20-%20New). This playbook automatically creates WAF policy custom rules to block the source IPs of the attacker as detected by the WAF analytics detection rules.